qiskit/visualization/pulse/matplotlib.py

```diff
@@ -596,7 +606,7 @@ def draw(self, schedule, dt, interp_method, plot_range,
 
         y0 = self._draw_channels(ax, output_channels, interp_method,
                                  t0, tf, dt, v_max, label=label,
-                                 framechange=framechange)
+                                 framechange=framechange, scaling=scaling)
```

Comment:
no `scaling` in `_draw_channels` args.
https://github.com/Qiskit/qiskit-terra/blob/a934b1f948854b754c0e9518f307ab3d0f6daa6b/qiskit/visualization/pulse/matplotlib.py#L489-L490